### PR TITLE
[self-hosted] Give hint about an error I got

### PR DESCRIPTION
**Pull request by jankeromnes**

### CODE REVIEW

**Comment by jankeromnes** [comment]
Actually doing just `helm delete gitpod` doesn't seem to fix this problem entirely.

When I try the command again after it, it now fails with:

```
$ helm upgrade --install $(for i in $(cat configuration.txt); do echo -e "-f $i"; done) gitpod .
Release "gitpod" does not exist. Installing it now.
Error: rendered manifests contain a resource that already exists. Unable to continue with install: existing resource conflict: kind: PersistentVolumeClaim, namespace: default, name: db-persistentvolumeclaim
```

File: src/docs/self-hosted/0.3.0/install/10_install_on_kubernetes.md [hunk 1]
```diff
@@ -62,8 +62,10 @@ helm upgrade --install $(for i in $(cat configuration.txt); do echo -e "-f $i";
 Visit `https://<your-domain.com>/` and check that you can login and start workspaces just like on gitpod.io.
 Launch a workpace. Launching the first workspace can take significantly longer (up to 15min), because docker images are pulled.
 
+Hint: If the install command fails with `Error: UPGRADE FAILED: "gitpod" has no deployed releases`, simply delete the existing installation with `helm delete gitpod` and try again.
```
Comment (32leaves) [inline src/docs/self-hosted/0.3.0/install/10_install_on_kubernetes.md]: maybe a `helm rollback` would solve this issue instead. With `helm 2` the deletion method worked reliably. Not sure that's still the case with helm 3.

**Comment by jankeromnes** [comment]
Actually, simply trying again seems to have fixed it:

```
$ helm upgrade --install $(for i in $(cat configuration.txt); do echo -e "-f $i"; done) gitpod .
Release "gitpod" does not exist. Installing it now.
NAME: gitpod
LAST DEPLOYED: Tue Dec 17 19:22:42 2019
NAMESPACE: default
STATUS: deployed
REVISION: 1
TEST SUITE: None
```

Now (almost) all the pods seem to be running fine:

```
$ kubectl get pods
NAME                                READY   STATUS                   RESTARTS   AGE
dashboard-7c55c958d9-wsczx          1/1     Running                  0          90s
db-5c9b9988df-jv9st                 1/1     Running                  0          90s
image-builder-5786c9664-46l24       2/2     Running                  0          90s
messagebus-6ddbf744db-4pgwl         1/1     Running                  0          90s
minio-6684dbbb9-7f4w5               1/1     Running                  0          89s
node-daemon-22sjz                   0/1     Init:RunContainerError   1          90s
proxy-56c6d9565b-4l46l              1/1     Running                  0          89s
registry-76dbddbf7d-2lfp6           1/1     Running                  0          90s
server-55cb99c7bb-krqzk             1/1     Running                  0          90s
ws-manager-8fbcf4cc4-grht4          1/1     Running                  0          89s
ws-manager-bridge-d49489859-pp9pn   1/1     Running                  0          89s
ws-sync-5ttg5                       1/1     Running                  0          90s
```

**Comment by svenefftinge** [comment]
@jankeromnes @32leaves is this still relevant?

**Comment by jankeromnes** [comment]
I would have found the hint useful, but I don't know if anyone else will run into the same issue. Maybe let's close the PR, and revive it if someone else hits this particular error.

**Pull request closed without merging**